app/javascript/mobile/foremMobile.js

```diff
@@ -66,7 +66,7 @@ export function foremMobileNamespace() {
       try {
         if (Runtime.isNativeIOS(namespace)) {
           window.webkit.messageHandlers[namespace].postMessage(message);
-        } else if (Runtime.isNativeAndroid(namespace)) {
+        } else if (Runtime.isNativeAndroid(`${namespace}Message`)) {
           AndroidBridge[`${namespace}Message`](JSON.stringify(message));
```

Comment:
This was the error that broke the native bridge messages (web -> native) for Android. The `isNativeAndroid` was returning `false`. With this fix it now matches on both lines with the correct format: `<namespace>Message` (i.e. `podcastMessage`, `userLoginMessage`, etc)